.github/workflows/compile.yaml

```
@@ -22,7 +22,7 @@ jobs:
         run: yum install -y selinux-policy-devel policycoreutils bzip2 perl
 
       - name: Compile policy
-        run: make DISTRO=rhel${{ matrix.centos }}
+        run: make DISTRO=rhel$(echo ${{ matrix.centos }} | sed 's/stream//')
```

You could also do
```suggestion
        run: make DISTRO=rhel$(rpm -E '%rhel')
```

Whether this is more readable or not, is up to you.

---

I like the decoupling of the two. Implemented.